civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -3993,75 +3317,6 @@ function civihr_employee_portal_schema_alter(&$schema) {
     'description' => 'Length of Service.',
   ];
 
-    // appraisal view:
-  $schema['appraisal']['description'] = 'Views data representation of CiviHR Appraisal';
```

Comment:
Please confirm that this will remove the view from the system, otherwise you'll need to add an upgrader for that too

Reply:
@ajesamson so, did you confirm that the view was removed?